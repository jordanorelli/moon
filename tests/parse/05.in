an_item: "this is the value of the item"
a_variable: an_item
