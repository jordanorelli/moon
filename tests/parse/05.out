root:
  assign:
    name:
      an_item
    value:
      string:
        this is the value of the item
  assign:
    name:
      a_variable
    value:
      variable:
        an_item
